packages/components/cascader/src/cascader.vue

```diff
@@ -248,6 +248,7 @@ import { ArrowDown, Check } from '@element-plus/icons-vue'
 import { cascaderEmits, cascaderProps } from './cascader'
 
 import type { Options } from '@element-plus/components/popper'
+import type { State } from '@popperjs/core'
```

Comment:
```suggestion
import type { Options, State } from '@element-plus/components/popper'
```
Put the `State` type and `Options` type together.